server/service/integration_mdm_test.go

```diff
@@ -6230,6 +6251,54 @@ func (s *integrationMDMTestSuite) assertConfigProfilesByIdentifier(teamID *uint,
 	return profile
 }
 
+func (s *integrationMDMTestSuite) assertMacOSConfigProfilesByName(teamID *uint, profileName string, exists bool) {
```

Comment:
Also copied / adjusted from Sarah's PR. Hopefully this will go smoothly for the merge in `main` conflict-wise.